### DISCUSSION TITLE
How do you query this data?

### DISCUSSION THREAD

**Discussion by radiantone** [martin-helmich/prometheus-nginxlog-exporter#233] [General]
Hi,
  I'm using grafana with prometheus and am not sure how to query this data. README doesn't explain.

thanks

**Comment by radiantone** [comment]
I found the docs for querying, and I run the exporter, but I don't see the metrics in prometheus or showing up in grafana

```
$ sudo ./prometheus-nginxlog-exporter -config-file  balsam.hcl -namespace balsam
loading configuration file balsam.hcl
&{{4040 0.0.0.0} {false     {  []}} [{balsam [/var/log/nginx/balsam_app.access.log] $remote_addr - - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" map[app:balsam environment:production]}]}
using configuration {{%!s(int=4040) 0.0.0.0} {%!s(bool=false)     {  []}} [{balsam [/var/log/nginx/balsam_app.access.log] $remote_addr - - [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" map[app:balsam environment:production]}]}
starting listener for namespace balsam
running HTTP server on address 0.0.0.0:4040
```
and grafana query
```sum(rate(balsam_http_response_size_bytes[5m])) by (instance)```

Nothing shows

**Reply by martin-helmich** [reply]
Sorry for the late response. Have you checked that your exporter actually exports metrics (by `GET`ting the `/metrics` endpoint on the listening address)? Also, did you configure your Prometheus instance to actually scrape the metrics from the exporter?

**Comment by radiantone** [comment]
Prometheus is running in a docker and exporter is not, btw.
